tests/phpunit/tests/term/meta.php

```diff
@@ -16,6 +16,7 @@ class Tests_Term_Meta extends WP_UnitTestCase {
 	public function set_up() {
 		parent::set_up();
 		register_taxonomy( 'wptests_tax', 'post' );
+		$this->reset_lazyload_queue();
```

Comment:
I don't understand why this is needed, the code in the default tearDown should ensure that the queues are empty.